scripts/benchmark/run.py

```diff
@@ -0,0 +1,275 @@
+import argparse
+import logging
+from pathlib import (
+    Path,
+)
+import sys
+import time
+
+import numpy
+
+from wasm import (
+    Runtime,
+)
+
+# From https://github.com/alexcrichton/rust-wasm-benchmark/blob/master/raw-wasm/raw.wast
+BASE_DIR = Path(__file__).parent
+
+BASIC_WASM = BASE_DIR / 'bench.wasm'
+
+FACTORIZATION_1117_WASM = BASE_DIR / 'factorization_1117_reikna_wasm.wasm'
+FACTORIZATION_9973_WASM = BASE_DIR / 'factorization_9973_reikna_wasm.wasm'
+FACTORIZATION_2147483647_WASM = BASE_DIR / 'factorization_2147483647_reikna_wasm.wasm'
+
+FIBONACCI_38_WASM = BASE_DIR / 'fibonacci_38_bigint_wasm.wasm'
+FIBONACCI_17_WASM = BASE_DIR / 'fibonacci_17_bigint_wasm.wasm'
+FIBONACCI_11_WASM = BASE_DIR / 'fibonacci_11_bigint_wasm.wasm'
+FIBONACCI_1_WASM = BASE_DIR / 'fibonacci_1_bigint_wasm.wasm'
+
+RECURSIVE_1_KECCAK = BASE_DIR / 'recursive_keccak_1_wasm.wasm'
+RECURSIVE_10_KECCAK = BASE_DIR / 'recursive_keccak_10_wasm.wasm'
+RECURSIVE_100_KECCAK = BASE_DIR / 'recursive_keccak_100_wasm.wasm'
+
+
+logging.basicConfig(level=logging.INFO, format='%(message)s', stream=sys.stdout)
+
+
+def _get_fn_address(runtime, module, function_name):
+    for export in module.exports:
+        if export.is_function and export.name == function_name:
+            return export.function_address
+    else:
+        raise Exception(f"No function found by name: {function_name}")
+
+
+def do_basic_benchmarks():
+    runtime = Runtime()
+    module, _ = runtime.instantiate_module(runtime.load_module(BASIC_WASM))
```

Comment:
Worth benchmarking load time?

Reply:
Yes, will do in another PR

tracking issue: https://github.com/ethereum/py-wasm/issues/113